v2_v3_outputs_map.csv

```diff
@@ -274,11 +271,6 @@ PV,pv_name,PV,name
 PV,size_kw,PV,size_kw
 PV,year_one_energy_produced_kwh,PV,year_one_energy_produced_kwh
 PV,year_one_energy_produced_bau_kwh,PV,year_one_energy_produced_kwh_bau
-PV,sr_provided_series_kw,PV (not in v3),
-PV,sr_required_series_kw,PV (not in v3),
-PV,station_distance_km,PV (not in v3),
-PV,station_latitude,PV (not in v3),
-PV,station_longitude,PV (not in v3),
 PV,year_one_power_production_series_kw,PV (not in v3),

```

Comment:
@gellwood year_one_power_production_series_kw should show as "(Removed in v3)"

Reply:
Also, one PV output is missing (a new output in v3): production_factor_series

Reply:
@adfarth I just committed a new version of the outputs spreadsheet. Hopefully, I did it correctly. 

I addressed these comments as well as the duplicate from the above comment.

Here's the link https://github.com/NREL/REopt-Analysis-Scripts/pull/112/commits/8ce15b57900064f546228e254684d37bf4f0a98b